src/docs/development/accessibility-and-localization/internationalization.md

```diff
@@ -364,7 +364,7 @@ class DemoApp extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return MaterialApp(
-       localeResolutionCallback(Locale locale, Iterable<Locale> supportedLocales) {
+       localeResolutionCallback:(Locale locale, Iterable<Locale> supportedLocales) {
```

Comment:
Thanks for catching this! There should be a space in between the `:` and `(Locale locale,...)`